### PR TITLE
C API instantiate with resolving imported functions

### PR DESCRIPTION
**Pull request by gumb0**

### CODE REVIEW

File: include/fizzy/fizzy.h [hunk 1]
```diff
@@ -71,6 +71,9 @@ struct FizzyModule* fizzy_parse(const uint8_t* wasm_binary, size_t wasm_binary_s
 /// Should be called unless @p module was passed to fizzy_instantiate.
 void fizzy_free_module(struct FizzyModule* module);
 
+bool fizzy_find_exported_function(
+    const struct FizzyModule* module, const char* name, uint32_t* out_func_idx);
```
Comment (axic) [inline include/fizzy/fizzy.h:75]: If this is done on the module that means the caller needs to do this prior to instantiation.

Comment (gumb0) [reply include/fizzy/fizzy.h:75]: I thinks it should be possible to access a module from an instance.

But C++ API has to be changed for that https://github.com/wasmx/fizzy/pull/533#discussion_r498271359

Comment (gumb0) [reply include/fizzy/fizzy.h:75]: Added now `fizzy_get_instance_module`

Comment (gumb0) [reply include/fizzy/fizzy.h:75]: Now I'm not sure again whether it would better to have an instance argument here.

**Comment by axic** [comment]
Can you create a new PR to move on from this branch name?

**Comment by gumb0** [comment]
Replaced by #608

**Pull request closed without merging**